CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [1.2.0] - FIXME
+- User-Agent header now reports `uploadcare-rails` and `rails` versions (overridable via config)
```

Comment:
@dmitry-mukhin I want to change this string to "Gem now reports us your `uploadcare-rails` and `rails` versions using the User-Agent header (overridable via config)" because it seems more informative and less confusing

Reply:
ok